RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h

```diff
@@ -33,6 +33,8 @@
 #include "TObject.h"
 #include "TH1F.h"
 
+#define MAX_VERTICES 200
```

Comment:
https://cms-cpt-software.web.cern.ch/cms-cpt-software/General/CodingAndStyleRules.pdf
for reco tests we have a link at 
https://twiki.cern.ch/twiki/bin/viewauth/CMS/RecoIntegration#For_reco_contact
the rules are linked in other places as well